packages/vitest/src/node/cli/cli-config.ts

```diff
@@ -0,0 +1,551 @@
+  },
+  middlewareMode: null,
+})
+
+const poolThreadsCommands: CLIOptions<ThreadsOptions & WorkerContextOptions> = {
+  isolate: {
+    description: 'Isolate tests in threads pool (default: true)',
+  },
+  singleThread: {
+    description: 'Run tests inside a single thread (default: false)',
+  },
+  maxThreads: {
+    description: 'Maximum number of threads to run tests in',
+    argument: '<workers>',
+  },
+  minThreads: {
+    description: 'Minimum number of threads to run tests in',
+    argument: '<workers>',
+  },
+  useAtomics: {
+    description: 'Use Atomics to synchronize threads. This can improve performance in some cases, but might cause segfault in older Node versions (default: false)',
+  },
+  execArgv: null,
+}
+
+const poolForksCommands: CLIOptions<ForksOptions & WorkerContextOptions> = {
+  isolate: {
+    description: 'Isolate tests in threads pool (default: true)',
+  },
+  singleFork: {
+    description: 'Run tests inside a single child_process (default: false)',
+  },
+  maxForks: {
+    description: 'Maximum number of processes to run tests in',
+    argument: '<workers>',
+  },
+  minForks: {
+    description: 'Minimum number of processes to run tests in',
+    argument: '<workers>',
+  },
+  execArgv: null,
+}
+
+function transformNestedBoolean(value: unknown) {
+  if (typeof value === 'boolean')
+    return { enabled: value }
+  return value
+}
+
+export const cliOptionsConfig: VitestCLIOptions = {
+  root: {
+    description: 'Root path',
+    shorthand: 'r',
+    argument: '<path>',
+    normalize: true,
+  },
+  config: {
+    shorthand: 'c',
+    description: 'Path to config file',
+    argument: '<path>',
+    normalize: true,
+  },
+  update: {
+    shorthand: 'u',
+    description: 'Update snapshot',
+  },
+  watch: {
+    shorthand: 'w',
+    description: 'Enable watch mode',
+  },
+  testNamePattern: {
+    description: 'Run tests with full names matching the specified regexp pattern',
+    argument: '<pattern>',
+    shorthand: 't',
+  },
+  dir: {
+    description: 'Base directory to scan for the test files',
+    argument: '<path>',
+    normalize: true,
+  },
+  ui: {
+    description: 'Enable UI',
+  },
+  open: {
+    description: 'Open UI automatically (default: !process.env.CI)',
+  },
+  api: {
+    argument: '[port]',
+    description: `Specify server port. Note if the port is already being used, Vite will automatically try the next available port so this may not be the actual port the server ends up listening on. If true will be set to ${defaultPort}`,
+    subcommands: apiConfig(defaultPort),
+  },
+  silent: {
+    description: 'Silent console output from tests',
+  },
+  hideSkippedTests: {
+    description: 'Hide logs for skipped tests',
+  },
+  reporters: {
+    alias: 'reporter',
+    description: 'Specify reporters',
+    argument: '<name>',
+    subcommands: null, // don't support custom objects
+  },
```

Comment:
I was testing multiple reporter options like this documentation https://vitest.dev/guide/reporters.html#combining-reporters and I hit the error:

```
$ pnpm -C test/core test basic.test.ts -- --reporter=json --reporter=default

> @vitest/test-core@ test /home/hiroshi/code/others/vitest/test/core
> vitest "basic.test.ts" "--reporter=json" "--reporter=default"

file:///home/hiroshi/code/others/vitest/packages/vitest/dist/cli.js:1208
      throw new Error(`Expected a single value for option "${command}"`);
            ^

Error: Expected a single value for option "--reporter <name>"
    at transform (file:///home/hiroshi/code/others/vitest/packages/vitest/dist/cli.js:1208:13)
    at setDotProp (file:///home/hiroshi/code/others/vitest/packages/vitest/dist/cli.js:258:22)
    at CAC.mri (file:///home/hiroshi/code/others/vitest/packages/vitest/dist/cli.js:642:9)
    at CAC.parse (file:///home/hiroshi/code/others/vitest/packages/vitest/dist/cli.js:559:27)
    at file:///home/hiroshi/code/others/vitest/packages/vitest/dist/cli.js:1342:13
    at ModuleJob.run (node:internal/modules/esm/module_job:218:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)
    at async main (file:///home/hiroshi/code/others/vitest/packages/vitest/dist/cli-wrapper.js:45:5)
```

Should we add `array: true` here to support this use case?

Reply:
Yes! And we can add a test for this 👍🏻